blazor/datagrid/template-editing.md

```diff
@@ -513,4 +513,172 @@ This is demonstrated in the following sample code, where the first input element
 ```
 
 The following image represents the AutoComplete component in focused state inside the dialog template of the DataGrid component,
-![Blazor DataGrid displays Dynamic Focus of Components](./images/blazor-datagrid-dynamic-focus-component.png)
\ No newline at end of file
+![Blazor DataGrid displays Dynamic Focus of Components](./images/blazor-datagrid-dynamic-focus-component.png)
+
+### Use FileUploader in Grid dialog edit template
+
+You can upload an image while adding or editing the column and show that image in the Grid column using the Column Template and Dialog Template features of the Grid. The Column Template feature is used to display the image in a Grid column, and the Dialog Template feature is used to render the SfUploader component for uploading the image while performing the dialog editing.
```

Comment:
To upload an image while adding or editing the column, show the image in Grid column using Column Template and Dialog Template features of the Grid. The Column Template feature is used to display image in a Grid column, and Dialog Template feature is used to render the SfUploader component for uploading the image while performing dialog editing.